### DISCUSSION TITLE
What is difference between stig-manager and stig-manager/rc containers?

### DISCUSSION THREAD

**Discussion by Rcline45** [NUWCDIVNPT/stig-manager#1158] [General]
I've notice there are multiple releases of stig-manager on ironbank. Is there any info on the differences between regular stig-manager, node-slim, and rc?

Thanks

**Comment by cd-rite** [comment]
Hi @Rcline45 
99.99% of the time, the baseline stig-manager and the rc images are identical. I reserved the "rc" name just in case we ever need to offer a "release candidate" version with substantial changes that we want to vet with the community before the mainline release. We have not had to do this yet, since most of our updates are small and incremental. 
The node-slim version is the same application , but based on a stripped-down "slim" version of the Iron Bank Node Base Image.  It is very similar to the `node:lts-alpine` based image we offer on Docker Hub.  Due to a bug in the base image, however, it does not currently support using external certificates from a volume (You could include them by building your own image based on that image, however). This *should* be resolved in Node 20, which we will move to when Iron Bank offers a Base Image of that version.

Thanks for the question, I'll try to update the README files on those repos to better express their intended purposes.
